docker-compose-elastic-cloud.yml

```diff
@@ -23,6 +23,11 @@ services:
       - ELASTIC_APM_SOURCE_LINES_SPAN_APP_FRAMES=5
       - ELASTIC_APM_SOURCE_LINES_ERROR_LIBRARY_FRAMES
       - ELASTIC_APM_SOURCE_LINES_SPAN_LIBRARY_FRAMES
+      - ELASTIC_APM_ENVIRONMENT=production
+      - ELASTIC_APM_ENABLE_LOG_CORRELATION=true
+      - ELASTIC_APM_LOG_LEVEL=DEBUG
+      - ENABLE_JSON_LOGGING=true
+      - ELASTIC_APM_USE_STRUCTLOG=true
```

Comment:
FYI, Both of these are already set to `True` in the Dockerfile

Reply:
removed form compose